win_installer.py

```diff
@@ -265,6 +291,24 @@ def unzip_file(filename, target_directory):
         z.extractall(target_directory)
 
 
+def vendor_in_from_python(package_name, destination_dir):
+    """
+    From the current Python environment, vendor in a stdlib package
+    which is not present in the embedded zip
+
+    FIXME: ideally this should actually involve copying a version
+    which matches the version of embedded Python we're using. Maybe
+    download the current .zip?
+    """
+    package = __import__(package_name)
+    for path in package.__path__:
+        shutil.copytree(
+            path,
+            os.path.join(destination_dir, package_name),
+            dirs_exist_ok=True,
```

Comment:
This argument was added in 3.8 addition, which is the reason AppVeyor is currently failing: 
https://docs.python.org/3/library/shutil.html#shutil.copytree

Reply:
Thanks, @carlosperate -- I hadn't spotted that.